gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStreamIntegrationTest.java

```diff
@@ -79,14 +94,26 @@ public void read_singleBytes() throws Exception {
     byte[] value = new byte[2];
     byte[] expected = Arrays.copyOf(testContent.getBytes(StandardCharsets.UTF_8), 2);
 
+    // enabled trace logging
     GoogleCloudStorageReadOptions options =
-        ghfs.getGcsFs().getOptions().getCloudStorageOptions().getReadChannelOptions();
+        GoogleCloudStorageReadOptions.builder().setTraceLogEnabled(true).build();
     FileSystem.Statistics statistics = new FileSystem.Statistics(ghfs.getScheme());
     try (GoogleHadoopFSInputStream in =
         new GoogleHadoopFSInputStream(ghfs, path, options, statistics)) {
       assertThat(in.read(value, 0, 1)).isEqualTo(1);
       assertThat(statistics.getReadOps()).isEqualTo(1);
+
+      assertingHandler.assertLogCount(1);
+      assertingHandler.verifyReadAPILogFields(GoogleHadoopFSInputStream.READ_METHOD);
+      assertingHandler.flush();
+
       assertThat(in.read(1, value, 1, 1)).isEqualTo(1);
+      // Total 4 calls for positional read
+      // 1. positional Read call
+      // 2. Seek to desired position
+      // 3. actual read from buffer after seek
+      // 4. Seek to old position
+      assertingHandler.assertLogCount(4);
```

Comment:
Do we want to validate the size read and the fact that logs for different APIs are logged?